### PR TITLE
Publish updated SharingService to allow creating persistent sessions

### PR DESCRIPTION
**Pull request by matthejo**
This change includes rebuild binaries from https://github.com/Microsoft/HoloToolkit/pull/83 to allow the SharingService to create persistent sessions.

### CODE REVIEW

**Comment by StephenHodgson** [comment]
Awesome update @matthejo.
Any chance you could provide example of usage in one of the sharing test scenes please.
Maybe we can add to the Sharing Manager in the editor so people can update the session type.

File: Assets/HoloToolkit/Sharing/Scripts/SDK/DiscoveryClientAdapter.cs [hunk 1]
```diff
@@ -2,6 +2,9 @@
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
 using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
```
Comment (StephenHodgson) [inline Assets/HoloToolkit/Sharing/Scripts/SDK/DiscoveryClientAdapter.cs]: Nit: did this file really require these usings?

Comment (matthejo) [inline Assets/HoloToolkit/Sharing/Scripts/SDK/DiscoveryClientAdapter.cs]: It doesn't, but the source files in the HoloToolkit repo still have these usings (and I used a script to copy everything over from there to the Unity project).  I can remove those there and then revert this part of the change in HoloToolkit-Unity.

**Comment by StephenHodgson** [comment]
https://github.com/Microsoft/HoloToolkit-Unity/issues/540

**Comment by NeerajW** [comment]
@StephenHodgson looks like we should merge this and recompile binaries from native HoloToolkit repo with latest as well? Thoughts?

**Comment by StephenHodgson** [comment]
I can merge, but as far as recompiling the binaries, I believe you guys at MSFT have a nice automated system to do that with. @jwittner?

**Comment by jwittner** [comment]
I've always done it via the scripts embedded in the native HTK repo. Namely, [BuildAll](https://github.com/Microsoft/HoloToolkit/blob/master/Sharing/BuildAll.bat), [CopyToSDK](https://github.com/Microsoft/HoloToolkit/blob/master/Sharing/CopyToSDK.bat), and [DeploySDKToUnity](https://github.com/Microsoft/HoloToolkit/blob/master/Sharing/DeploySDKToUnity.bat).

**Comment by StephenHodgson** [comment]
Oh, in that case, should be fairly straight forward.

**Comment by DDReaper** [comment]
ANy progress on this @StephenHodgson ? Given your comments on the Sharing framework, should this be closed or merged now?

**Comment by keveleigh** [comment]
I'm planning on signing/updating all MixedRealityToolkit binaries next week.

**Comment by StephenHodgson** [comment]
I'll leave that up to you Kev

**Comment by StephenHodgson** [comment]
Should we merge or close this @keveleigh?

**Comment by DDReaper** [comment]
Any update on this PR @keveleigh / @StephenHodgson ?
Don't like long in the tooth PR's :P

**Comment by StephenHodgson** [comment]
Duplicate of #1562.  Going to go ahead and close this

**Pull request closed without merging**